functions/.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Should we have .vscode in both root and this folder?

Reply:
Good catch, was auto generated i have ```.vscode``` hidden in my explorer.